src/samples/AppSelector/index.tsx

```diff
@@ -19,32 +16,11 @@ import ProofOfEntitlement from '../ProofOfEntitlement/ProofOfEntitlement';
 import ChangeOfBank from '../ChangeOfBank/ChangeOfBank';
 
 const AppSelector = () => {
-  const [i18nloaded, seti18nloaded] = useState(false);
-
   useEffect(() => {
-    i18n
-      .use(Backend)
-      .use(initReactI18next)
-      .init({
-        lng: sessionStorage.getItem('rsdk_locale')?.substring(0, 2) || 'en',
-
-        backend: {
-          loadPath: `assets/i18n/{{lng}}.json`
-        },
-        fallbackLng: 'en',
-        debug: false,
-        returnNull: false,
-        react: {
-          useSuspense: false
-        }
-      })
-      .finally(() => {
-        seti18nloaded(true);
-        setPageTitle();
-      });
```

Comment:
Loaded i18n in rather than in the useEffect - https://react.i18next.com/getting-started#basic-sample

Also displaying null while it was loading has been cleaned up too